typings.json

```diff
@@ -8,7 +8,6 @@
     "merge-stream": "registry:dt/merge-stream#1.0.0+20160313224417",
```

Comment:
Only DT stuff in their 2.0 branch is in `@types`. They're using this as a chance to standardize on modern practices.